packages/invoices/src/OutboundShipment/DetailView/modals/ItemDetailsModal.tsx

```diff
@@ -83,7 +83,16 @@ const listQueryFn = async (): Promise<Item[]> => {
   return items.nodes;
 };
 
-const sortByExpiryDate = (a: BatchRow, b: BatchRow) => {
+const sortByDisabledThenExpiryDate = (a: BatchRow, b: BatchRow) => {
```

Comment:
it was another suggestion - put the disabled rows last

Reply:
Ah right, I was a little confused by the order before seeing this.

I think something like a different colour for hold and 0 quantity (so, 3 different colours) or indicator.. maybe the on hold rows have very little opacity and 0quant rows have the quantity column with a background or something.. and then an order like: 0>Quantity, <=0 Quantity, On Hold

hehe

Reply:
good idea - and good feedback that the strange ordering is not clear!